usr/initiator.c

```diff
@@ -588,7 +591,7 @@ __session_conn_reopen(iscsi_conn_t *conn, queue_task_t *qtask, int do_stop,
 	iscsi_session_t *session = conn->session;
 	uint32_t delay;
 
-	log_debug(1, "re-opening session %d (reopen_cnt %d)", session->id,
+	log_session_reopen(1, session, "re-opening session %d (reopen_cnt %d)", session->id,
```

Comment:
so instead of logging this message if debug logging is enabled at all, now it prints it every time, by default (since freq=1 by default)?

Reply:
Changed.